hazelcast/src/main/java/com/hazelcast/map/impl/recordstore/expiry/ExpirySystemImpl.java

```diff
@@ -173,6 +174,11 @@ public final void add(Data key, long ttl, long maxIdle,
         storeExpiryMetadata(key, ttl, maxIdle, expiryTime, lastUpdateTime);
     }
 
+    @Override
+    public final void add(Data key, long lastUpdateTime, long now) {
+        add(key, UNSET, UNSET, UNSET, lastUpdateTime, now);
```

Comment:
I'm thinking, maybe it is better to directly write the implementation of this new method than bring a new dependency with UNSET. 
Because it is not clear, what logic should be executed in the case of those UNSETs.